src/Workflow/ProcessWorkflow.php

```diff
@@ -81,9 +84,10 @@ public static function getWorkflow(EventDispatcherInterface $eventDispatcher): W
     public static function getTransitions(): array
     {
         return [
-            new Transition(self::TRANSITION_START, [ImageInterface::PROCESSING_STATE_PENDING, ImageInterface::PROCESSING_STATE_FAILED], ImageInterface::PROCESSING_STATE_PROCESSING),
+            new Transition(self::TRANSITION_ENQUEUE, [ImageInterface::PROCESSING_STATE_INITIAL, ImageInterface::PROCESSING_STATE_FAILED, ImageInterface::PROCESSING_STATE_PROCESSED], ImageInterface::PROCESSING_STATE_PENDING),
+            new Transition(self::TRANSITION_START, ImageInterface::PROCESSING_STATE_PENDING, ImageInterface::PROCESSING_STATE_PROCESSING),
```

Comment:
Why should we be able to make this transition from the `processing` state?

Reply:
From the `processing` state? I don't see that specified in the code you comment on?

Reply:
Hmmmmmm! This GitHub interface is toying with me